services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_api/_core.py

```diff
@@ -201,17 +184,15 @@ async def _get_task_data_when_service_running(service_id: str) -> Mapping[str, A
         service_state = task["Status"]["State"]
 
         if service_state not in TASK_STATES_RUNNING:
-            raise TryAgain()
+            raise TryAgain
```

Comment:
I think I've already expressed my disagreement on this one. Why mix classes and objects. It makes more senso to raise instances of errors and not classes. At least it is consistent.

Reply:
see [https://beta.ruff.rs/docs/rules/unnecessary-paren-on-raise-exception/](https://beta.ruff.rs/docs/rules/unnecessary-paren-on-raise-exception/)
I disagree with you, but since it is your code I will revert it.